lints/lint_dnsname_bad_character_in_label.go

```diff
@@ -0,0 +1,69 @@
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+	"regexp"
+	"strings"
+)
+
+type DNSNameProperCharacters struct {
+	// Internal data here
+	CompiledExpression *regexp.Regexp
+}
+
+func (l *DNSNameProperCharacters) Initialize() error {
```

Comment:
The code base never actually checks whether this call returns an error.

Reply:
IMHO `Initialize()` should not return an error, similar to `func init()`. If some really bad happens, `panic()` internally. There's not much the framework could do with a non-nil error, anyway.

Reply:
I'm fine with that. We just gotta do something about it.

Reply:
`MustCompile` does exactly this. I'll change to that.